control-plane-operator/controllers/hostedcontrolplane/manifests/storage.go

```diff
@@ -0,0 +1,35 @@
+package manifests
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+func ClusterStorageOperatorDeployment(ns string) *appsv1.Deployment {
+	dep := &appsv1.Deployment{}
+	dep.Name = "cluster-storage-operator"
+	dep.Namespace = ns
+	return dep
+}
+
+func ClusterStorageOperatorRole(ns string) *rbacv1.Role {
+	role := &rbacv1.Role{}
+	role.Name = "cluster-storage-operator-role"
```

Comment:
name is redundant with type "cluster-storage-operator-role" -> "cluster-storage-operator"

Reply:
Renamed